src/main/java/org/benetech/mathshare/controllers/ProblemController.java

```diff
@@ -0,0 +1,41 @@
+package org.benetech.mathshare.controllers;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import org.benetech.mathshare.model.dto.ProblemSetDTO;
+import org.benetech.mathshare.service.ProblemService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/set")
+@SuppressFBWarnings(value = "SLF4J_FORMAT_SHOULD_BE_CONST",
+        justification = "We need Logger to contain not-constant data to be valuable")
```

Comment:
*not-constant -> non-constant
What do you mean by 'valuable'?

Reply:
'valuable', so it would give us information about the error, with the stack trace.
I'll change to non-constant, thanks